tock/hours/migrations/0066_auto_20221230_1357.py

```diff
@@ -0,0 +1,34 @@
+# Generated by Django 3.2.16 on 2022-12-20 18:16
+
+from django.db import migrations
+from django.conf import settings
+
+
+def calculate_total_weekly_allocation(apps, schema_editor):
+    Timecard = apps.get_model('hours', 'Timecard')
+    timecards = Timecard.objects.all()
+
+    for tc in timecards:
+        timecardobjs = tc.timecardobjects.all()
+        if len(timecardobjs) == 0:
+            continue
```

Comment:
If the timecard doesn't have any objects then it looks like the default for the new fields is null? I'm not sure what happens in this scenario -- did we want the default to be 0 instead?

Reply:
Oh yes! let's make the default `0`, good call.

Reply:
Great, thanks!